ts/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "grpc-web",
```

Comment:
can you name this `grpc-web-client` instead? so that we don't have a clash with the Google one in the future?